canarytokens/channel_output_email.py

```diff
@@ -376,6 +407,38 @@ def format_report_text(details: TokenAlertDetails, body_length: int = 999999999)
             ).strip()
         return body
 
+    @staticmethod
+    def format_token_exposed_text(
+        details: TokenExposedDetails, body_length: int = 999999999
+    ):
+        """Returns a string containing an incident report in text,
+        suitable for emailing"""
+        if body_length <= 140:
```

Comment:
When do we have a body length of less than 140? Just to understand why there are two options here.

Reply:
There aren't any cases in the code currently where `body_length` is shorter than `140` characters. It's probably left over from earlier code. I've removed `body_length` and the if statement (since `body_length` is never explicitly specified when calling `format_token*_text`, the default value is always used).